src/common/AnimatedPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSpring, animated } from 'react-spring';
+
+const AnimatedPage = ({ children }) => {
+    const props = useSpring({
+        opacity: 1,
+        from: { opacity: 0 },
+        config: {
+            duration: 800,
```

Comment:
I forgot to comment on that in previous review, please check if shorter duration ie. 400ms doesn't look better (it does to me)

Reply:
I've tested the shorter duration of 400ms for the opacity animation, and I agree with you; it does look better and adds a quicker, smoother transition. I've made the change as per your recommendation. 
Thanks for the suggestion! 😊